ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -130,7 +130,7 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     this.jupyterLabMode = jupyterLabMode === 'true';
     this.setNotebookNames();
 
-    this.loadingSub = this.clusterService.listClusters()
+    this.loadingSub = this.clusterService.listClusters(this.wsNamespace)
```

Comment:
we have access to a `cluster` model in this class which contains `clusterNamespace`. Based on the API naming, I would guess that `cluster.clusterNamespace` is the right value to use here. Do they just happen to the same in our business logic?

Reply:
That is the better choice.  Thanks.